Sources/Valkey/Subscriptions/ValkeySubscription.swift

```diff
@@ -51,8 +51,19 @@ public struct ValkeySubscription: AsyncSequence, Sendable {
     public struct AsyncIterator: AsyncIteratorProtocol {
         var base: BaseAsyncSequence.AsyncIterator
 
+        #if compiler(>=6.2)
+        @concurrent
```

Comment:
Could this be 
```swift
public mutating func next() async throws -> Element? {
    try await self.base.next(isolation: #isolation)
}
```
Instead of adding `@concurrent`